app/Http/Controllers/AccountController.php

```diff
@@ -42,10 +42,11 @@ public function update(Account $account)
     {
         if ($account->update(Request::all())) {
             return redirect(route('accounts.show', $account->id))->with('alerts.success', trans('crud.accounts.updated'));
-        }
+        }else{
```

Comment:
there is no need to use `else` statement, if the condition `return false`, it will go to the next line without execute the code inside the `if` statement